README.md

```diff
@@ -48,12 +48,7 @@ cd HandyRL
 
 Then, install additional libraries (e.g. numpy, pytorch). Or run it in a virtual environment or container (e.g. Docker).
 ```
-pip3 install -r requirements.txt
-```
-
-To use games of kaggle environments (e.g. Hungry Geese) you can install also additional dependencies.
-```
-pip3 install -r handyrl/envs/kaggle/requirements.txt
+poetry install
```

Comment:
IIUC, handyrl is not a library yet and intended to use with environments under `env/`. And, this is a framework of training. So we don't need to care about docker and additional dependency `poetry`.

Additional option is:

1. Provide both pyproject.toml and requirements.txt.
2. Check coherence of them in CI by using https://python-poetry.org/docs/cli/#export

Reply:
https://github.com/DeNA/HandyRL/pull/182#issuecomment-820229331

We provide requirements.txt until https://github.com/DeNA/HandyRL/issues/183 .